From a199503941db4cf6bdda1d14a2dc4964478f7dad Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:04:31 -0400
Subject: [kernel] perf/events/core: Drop unused variable after cleanup

Message-id: <1410181752-25631-27-git-send-email-jolsa@redhat.com>
Patchwork-id: 91702
O-Subject: [PATCH RHEL7.1 BZ1134356 026/307] perf/events/core: Drop unused variable after cleanup
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 12665b35b0b48c9583ee1b8f0a403dc708fb4a92
Author: Borislav Petkov <bp@suse.de>
Date: Sat May 10 13:10:59 2014 +0200

description
===========
... in 3a497f48637 ("perf: Simplify perf_event_exit_task_context()")
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/kernel/events/core.c b/kernel/events/core.c
index 1ce2fca..8d2d235 100644
--- a/kernel/events/core.c
+++ b/kernel/events/core.c
@@ -7447,7 +7447,7 @@ __perf_event_exit_task(struct perf_event *child_event,
 
 static void perf_event_exit_task_context(struct task_struct *child, int ctxn)
 {
- struct perf_event *child_event, *tmp;
+ struct perf_event *child_event;
  struct perf_event_context *child_ctx;
  unsigned long flags;
 
-- 
1.7.1